examples/real-world/src/root/routes.js

```diff
@@ -1,22 +1,26 @@
 import React from 'react'
 import { Route, Switch } from 'react-router' 
 import { SubspaceProvider } from 'react-redux-subspace'
+
+// work around for lerna/npm link issue for local development
+import { ReactReduxContext } from 'react-redux'
```

Comment:
You'll see this a lot in the review.  When `lerna` links the packages using `npm link` they end up with different instances of `react-redux` so the `React.createContext(...)` calls create a different context so `react-redux-subspace` cannot access the store from the examples `Provider`.  This follows the "BYO context` concept from `react-redux` but provides the actual context instance from the example's version.

This works, but it's a bit gross and not really how you would use in a real app, which makes having it in all the examples a bit annoying.

Reply:
This is one annoying thing about reacts new context api.

Could we have a post install script that removes the nested react-redux packages in node_modules so they all resolve to the same one?

At least then the fix is hidden away in the package.json and not in the example code.

Reply:
Actually, that would break the build of the actual package... Ignore that.

Reply:
The issue with that is the file is not in the same directory tree, so the node package resolution does not find the package at all. I have done a trick in the past to `npm link` the troublesome package back to the other's as a post-installation task, but I found it to be slow and required a bit of hacking about to get working.  At least this is actual functionality offered by both `react-redux` and now `redux-subspace`, it just sucks a bit that it's the defacto example.

Reply:
At least it won't break is hey do do it in production code (it'll be the same as the default value)

Maybe adjust the comment a bit to make it more obvious?

"// Note: please don't copy this line line (and associated prop usage) to your project, you are unlikely to need to do this for standard subspace usage.
// This is caused by Leena using  non link..."

Reply:
lol at autocorrect, but I like the idea.

Reply:
That'll teach me for reviewing on my phone.